en/simulation/simulation-in-hardware.md

```diff
@@ -45,13 +45,13 @@ The simulated quadrotor can be displayed in jMAVSim from PX4 v1.11.
 1. Unplug and replug the hardware autopilot (allow a few seconds for it to boot).
 1. Start jMAVSim by calling the script **jmavsim_run.sh** from a terminal:
    ```
-   ./Tools/jmavsim_run.sh -q -d /dev/ttyACM0 -b 921600 -r 250 -o
+   ./Tools/jmavsim_run.sh -q -d /dev/ttyACM0 -b 2000000 -r 250 -o
```

Comment:
@julianoes I'm OK with this and will merge, though it would be good to understand why. 

Question - here you update SIH (not SITL). I assume it applies here too?